CMakeLists.txt

```diff
@@ -191,8 +191,15 @@ rocm_create_package(
 set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/lib)
 set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/lib)
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/bin)
-
 add_subdirectory(src)
 add_subdirectory(doc)
 add_subdirectory(test)
 add_subdirectory(tools)
+
+set(DEST_DIR ${CMAKE_BINARY_DIR})
+file (COPY ${CMAKE_SOURCE_DIR}/src/py/backend/backend.py DESTINATION ${DEST_DIR}/lib/onnx_migraphx/.)
+file (COPY ${CMAKE_SOURCE_DIR}/src/py/backend/backend_rep.py DESTINATION ${DEST_DIR}/lib/onnx_migraphx/.)
+file (COPY ${CMAKE_SOURCE_DIR}/src/py/backend/__init__.py DESTINATION ${DEST_DIR}/lib/onnx_migraphx/.)
+file (COPY ${CMAKE_SOURCE_DIR}/test/py/onnx_backend_test_series.py DESTINATION ${DEST_DIR}/.)
+file (COPY ${CMAKE_SOURCE_DIR}/test/py/testdata DESTINATION ${DEST_DIR}/.)
```

Comment:
`configure_file` should probably be used instead so that cmake will re copy this when the file changes.

Although, is it possible to achieve this without copying? What about naming the directory `py/onnx_migraphx`?

Reply:
ok, will change to that.

Reply:
After trying the above, we get the following error as:
  File "/home/scxiao/anaconda3/lib/python3.6/site-packages/_pytest/config.py", line 13, in <module>
    import _pytest._code
  File "/home/scxiao/anaconda3/lib/python3.6/site-packages/_pytest/_code/__init__.py", line 3, in <module>
    from .code import Code  # noqa
  File "/home/scxiao/anaconda3/lib/python3.6/site-packages/_pytest/_code/code.py", line 11, in <module>
    reprlib = py.builtin._tryimport('repr', 'reprlib')
AttributeError: module 'py' has no attribute 'builtin'

Looks like the pytest module contains something conflict.